### ISSUE TITLE
Problem with Delete Command

### ISSUE THREAD

**Issue by Michi-98** [ioBroker/ioBroker.influxdb#236]
.Hey guys,
I recently started using the new Delete command, but it always causes me problems.
The command deletes the data from the database as intended, but I always get the following error message:
`Error on delete("DELETE FROM "measurement" WHERE time = '2022-03-14T13:38:17.332Z'): Error: shard 126: short write / "shard 126: short write"".`

I don't know if this is related. But since I've been using the delete function, only the data points where I've already deleted something always get this error message when I want to save more data:
```
2022-03-14 00:00:00.857 - info: influxdb.0 (2254) Add point that had error for measurement to buffer again, error-count=4
2022-03-14 00:00:00.868 - warn: influxdb.0 (2254) Error on writePoint("{"value":103.52, "time": "2022-03-13T23:00:00.128Z", "from": "system.adapter.javascript.0", "q":0, "ack":true}): Error: {"error": "short write"}
```

To be able to use the datapoints again for saving, I have to restart the whole system and reconfigure the datapoints.

**Comment by Apollon77** [comment]
In fact the error comes from Influxdb ... google brought me to https://community.influxdata.com/t/error-shard-num-short-write/17597 ... maybe that helps? But in fact nothing where the adapter can do anything

**Comment by Michi-98** [comment]
Hello,
thanks to you @Apollon77 I was able to solve the problem.
In case others also have this problem, I would like to post the solution here.
Indeed, the problem seems to appear only when saving the data with the new "TSI1" instead of the "inmem" method.
There must have been a synchronization problem between the TSI shards and the .tsm files.

1. To solve this, I first stopped the influx server. On Linux the following command:
`service influxdb stop`
2. After that I generated new TSI shards using the .tsm files. The command is
`influx_inspect buildtsi -datadir <direction of data> -waldir <direction of wal>`
Default is <Direction of Data> = /var/lib/influxdb/data and <Direction of wal> = /var/lib/influxdb/wal
3. Start Influxserver again with
`service influxdb start`

I hope that I could help more people with this.
